components/ws-daemon/pkg/controller/workspace_controller.go

```diff
@@ -316,6 +317,9 @@ func (wsc *WorkspaceController) handleWorkspaceStop(ctx context.Context, ws *wor
 			ws.Status.SetCondition(workspacev1.NewWorkspaceConditionBackupFailure(disposeErr.Error()))
 		} else {
 			ws.Status.SetCondition(workspacev1.NewWorkspaceConditionBackupComplete())
+			if ws.Spec.Type != workspacev1.WorkspaceTypeRegular {
+				ws.Status.Snapshot = snapshotUrl
+			}
```

Comment:
Tried commenting these changes out, and it seems we are still able to report prebuild statuses just fine 🤘